samples/meetings-details-tab/nodejs/server/api/homeController.js

```diff
@@ -1,29 +1,26 @@
 const configuration = require('dotenv').config();
 const store = require('../services/store')
 const {createAdaptiveCard }= require('../services/AdaptiveCardService')
-const env = configuration.parsed;
-const appID = env.BotId;
-const appPassword = env.BotPassword;
 const { ConnectorClient, MicrosoftAppCredentials } = require('botframework-connector');
-const credentials = new MicrosoftAppCredentials(appID,appPassword);
+const credentials = new MicrosoftAppCredentials(process.env.BotId,process.env.BotPassword);
 
  const sendAgenda = async (req) => {
     const data = req.body;
     store.setItem("agendaList", data.taskList);
-    const ConversationID = store.getItem("conversationId");
-    const ServiceUrl = store.getItem("serviceUrl");
-    const client = new ConnectorClient(credentials, { baseUri: ServiceUrl });
+    const conversationID = store.getItem("conversationId");
+    const serviceUrl = store.getItem("serviceUrl");
+    const client = new ConnectorClient(credentials, { baseUri: serviceUrl });
```

Comment:
this is still not changed, I can still see `baseUri` in connectorClient, I believe it should be renamed as serviceURL,